modules/security-hardening-what.adoc

```diff
@@ -6,13 +6,15 @@
 
 = Choosing what to harden in {op-system}
 ifdef::openshift-origin[]
-The link:https://docs.fedoraproject.org/en-US/Fedora/19/html/Security_Guide/chap-Security_Guide-Basic_Hardening.html[{op-system-base} Security Hardening] guide describes how you should approach security for any {op-system-base} system.
+The link:https://docs.redhat.com/en/documentation/red_hat_enterprise_linux/9#Security[Security] category in the Red Hat Enterprise Linux 9 documentation describes how you should approach security for any {op-system-base} system.
```

Comment:
```suggestion
To know how to approach security for any {op-system-base} system, see the link:https://docs.redhat.com/en/documentation/red_hat_enterprise_linux/9#Security[Security] category in the Red{nbsp}Hat Enterprise Linux 9 documentation.
```
This is up to you.
I was thinking about `"what" --> "how"` and avoiding `should`.